14_functions_lab/03.py

```diff
@@ -1,5 +1,14 @@
-"""
-Write a function that calculates the sum
-of the 10th digit from all arguments passed to it
-"""
-
+def TensSum(*list):

```

Comment:
Please leave the top comments in the file (the ones inside """). They make it easier for everyone opening the file later to understand what's inside